clink/lua/src/settings_api.cpp

```diff
@@ -74,7 +74,7 @@ static int get(lua_State* state)
 /// -name:  settings.set
 /// -ver:   1.0.0
 /// -arg:   name:string
-/// -arg:   value:string
+/// -arg:   value:boolean | string
```

Comment:
After deeper investigation, I think it's best to say either `string` or `boolean | integer | string`.

Although the `settings.set()` API does not accept an integer argument, Lua itself will internally convert an integer argument into a string argument when the API asks Lua to verify that the argument is a number.  So, the overall result is that the API effectively supports an integer `value` argument.